code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -304,6 +331,88 @@
 		busy = 0
 		stop_automated_movement = 0
 
+/mob/living/simple_animal/hostile/giant_spider/verb/web()
+	set name = "Spin Web"
+	set desc = "Create a web that slows down movement."
+	set category = "Greimorian"
+
+	var/obj/effect/spider/stickyweb/W = locate() in get_turf(src)
+	if(!W)
+		src.visible_message("\The [src] begins to secrete a sticky substance.")
+		if(!do_after(src, 10)) return
+		new /obj/effect/spider/stickyweb(src.loc)
+
+
+/mob/living/simple_animal/hostile/giant_spider/nurse/verb/cocoon()
+	set name = "Cocoon and Feed"
+	set desc = "Cocoon an incapacitated mob so you can feed upon it. This will give you one food point."
+	set category = "Greimorian"
+
+	for(var/mob/living/P in range(1,src))
+		cocoon_target = P
+		if(P.stat && !istype(P,/mob/living/simple_animal/hostile/giant_spider))
+			if(get_dist(src, cocoon_target) <= 1)
+				src.visible_message("\The [src] begins to secrete a sticky substance around \the [cocoon_target].")
+				if(!do_after(src, 80)) return
+				if(cocoon_target && istype(cocoon_target.loc, /turf) && get_dist(src,cocoon_target) <= 1)
+					var/obj/effect/spider/cocoon/C = new(cocoon_target.loc)
+					var/large_cocoon = 0
```

Comment:
this is a boolean, so change to TRUE/FALSE as needed

Reply:
Handled